src/extension/src/file_handlers/ExtOutputStatusHandler.py

```diff
@@ -138,10 +138,10 @@ def update_file(self, seq_no, status=Constants.Status.Transitioning.lower(), cod
             if status_json is None:
                 self.logger.log_error("Error processing file. [File={0}]".format(file_name))
                 return
-            self.update_key_value_safely(status_json, self.file_keys.status_status, status, self.file_keys.status_status)
-            self.update_key_value_safely(status_json, self.file_keys.status_code, code, self.file_keys.status_status)
-            self.update_key_value_safely(status_json, self.file_keys.timestamp_utc, str(datetime.datetime.utcnow().strftime(Constants.UTC_DATETIME_FORMAT)))
-            self.update_key_value_safely(status_json, self.file_keys.status_formatted_message_message, str(message), self.file_keys.status_formatted_message)
+            self.update_key_value_safely(status_json[0], self.file_keys.status_status, status, self.file_keys.status_status)
+            self.update_key_value_safely(status_json[0], self.file_keys.status_code, code, self.file_keys.status_status)
+            self.update_key_value_safely(status_json[0], self.file_keys.timestamp_utc, str(datetime.datetime.utcnow().strftime(Constants.UTC_DATETIME_FORMAT)))
+            self.update_key_value_safely(status_json[0][self.file_keys.status_status], self.file_keys.status_formatted_message_message, str(message), self.file_keys.status_formatted_message)
```

Comment:
should there be a comma between status_json[0] and self.file_keys?

Reply:
No because it's to access the key inside status_json[0]